absl/strings/numbers.h

```diff
@@ -24,7 +24,26 @@
 #define ABSL_STRINGS_NUMBERS_H_
 
 #ifdef __SSE4_2__
```

Comment:
This does not look correct to me. `__SSE4_2__` is not defined by MSVC.

How was this change tested?